state-sync/diem-data-client/src/lib.rs

```diff
@@ -307,6 +307,31 @@ pub struct AdvertisedData {
     pub transaction_outputs: Vec<CompleteDataRange<Version>>,
 }
 
+// impl fmt::Debug for AdvertisedData {
```

Comment:
My initial approach was implementing `Debug` directly for `AdvertisedData` instead of implementing `Debug` for `LedgerInfoWithV0` (as done below). My thinking was that this way would not affect anyone still wanting to see the full debug info of `LedgerInfoWithV0` struct. I'll remove this block later.

Reply:
I actually think that this is the right approach (see my comments below)